src/commands/notify.yml

```diff
@@ -24,18 +24,78 @@ parameters:
     description: Notification message
     type: string
   priority:
-    description: 'Notification priority (low-to-high: -2 to 1)'
+    description: "Notification priority (low-to-high: -2 to 1)"
     type: integer
     default: 0
+  fail_only:
+    description: If `true`, notifications successful jobs will not be sent
+    type: boolean
+    default: false
+  only_for_branches:
```

Comment:
Is there any benefit of this over using the built-in branch filtering?

For example:

```yaml
      - pushover/notify:
          name: notify
          requires: [release-frontend, release-backend]
          title: 'Restyled deployments'
          message: 'restyled.io deployed at ${CIRCLE_SHA1:0:10}'
          filters:  # <-- here
            branches:
              only:
                - master
```

Reply:
I want to be notified when a job I created fails. So it needs to be implemented in the command, not the job.

This is a use case:
https://github.com/masutaka/circleci-bundle-update-pr/blob/fd3401ec320a6b72de9a83a11f69d6d499318a5c/.circleci/config.yml

FYI: It can't specify `filters` in the CircleCi command.
https://circleci.com/docs/2.0/configuration-reference/#commands-requires-version-21